Wireframe/index.html

```diff
@@ -4,10 +4,52 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    
+      <header>
+        <h1>Understanding Git</h1>
+        <P>"Discover the basics of Git, a powerful tool for tracking code changes and teamwork. Learn why developers use Git and how branches make coding more organized and efficient."</P>
+      </header>
+
+
+      <main>
+
+        <section class="one">
+          <article>
+             <img src="images/git.jpg">
```

Comment:
```suggestion
             <img src="images/git.jpg" alt="">
```

Why do we add alt attributes to images? What would be a good value for this alt attribute I have selected?